server/src/test/java/io/spine/server/GrpcContainerTest.java

```diff
@@ -109,6 +110,38 @@ void startServerWithExecutor() throws IOException {
                 .isNotNull();
     }
 
+    @Test
+    @DisplayName("configure underlying gRPC server")
+    void configureUnderlyingGrpcServer() {
+        int port = 1654;
+        CommandService service = CommandService.newBuilder()
+                                               .build();
+        GrpcContainer container =
+                GrpcContainer.atPort(port)
+                             .apply((server) -> server.addService(service))
```

Comment:
Let's make the naming a bit more precise. 
```java
GrpcContainer.atPort(port)
             .withServer(s -> s.addService(service))
             .build();
```

Method `apply(..)` seems a bit too general. On another minor note, it's going to cause confusion on Kotlin and, probably, some extra syntax trickery to resolve the clash for the Kotlin compiler.